lib/ansible/modules/network/mlnxos/mlnxos_command.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+
+DOCUMENTATION = """
+---
+module: mlnxos_command
+extends_documentation_fragment: mlnxos
+version_added: "2.5"
+author: "Samer Deeb (@samerd)"
+short_description: Run commands on remote devices running MLNXOS
```

Comment:
Please update to:
`short_description: Run commands on remote devices running  Mellanox MLNX-OS`

We use company & product name so it's easier for people to find modules.